stl/inc/mdspan

```diff
@@ -705,6 +729,18 @@ public:
     using data_handle_type = typename accessor_type::data_handle_type;
     using reference        = typename accessor_type::reference;
 
+    static_assert(
+        sizeof(_ElementType) > 0, "ElementType must be a complete type (N4928 [mdspan.mdspan.overview]/2.1).");
+    static_assert(
+        !is_abstract_v<_ElementType>, "ElementType cannot be an abstract type (N4928 [mdspan.mdspan.overview]/2.1).");
+    static_assert(
+        !is_array_v<_ElementType>, "ElementType cannot be an array type (N4928 [mdspan.mdspan.overview]/2.1).");
+    static_assert(
+        _Is_extents<_Extents>, "Extents must be a specialization of extents (N4928 [mdspan.mdspan.overview]/2.2).");
+    static_assert(is_same_v<_ElementType, typename _AccessorPolicy::element_type>,
+        "Expression is_same_v<ElementType, typename AccessorPolicy::element_type> must be true (N4928 "

```

Comment:
Suggestion: This could be phrased more clearly as "X and Y must be the same type", which would follow the static assertions above that nicely translated `!is_abstract_v` into more human language. This occurs below for `is_constructible_v` assertions etc.